### PR TITLE
Add getNodeInfoByID and getSegmentInfoByNode  function for cluster

### PR DESCRIPTION
**Pull request by xige-16**
fix issue: #9481
Signed-off-by: xige-16 <xi.ge@zilliz.com>

### CODE REVIEW

**Comment by xige-16** [comment]
loadsegment reschedule should be refactored to genenrate watchDeltaChannelRequests

**Comment by xige-16** [comment]
seek position of delta channel should be save in segment checkout point

File: internal/querynode/impl.go [hunk 1]
```diff
@@ -559,33 +497,8 @@ func (node *QueryNode) GetSegmentInfo(ctx context.Context, in *queryPb.GetSegmen
...
-				}
-				return res, err
-			}
-			info := getSegmentInfo(segment)
-			log.Debug("QueryNode::Impl::GetSegmentInfo for streaming", zap.Any("SegmentID", id), zap.Any("info", info))
-			infos = append(infos, info)
-		}
-	}
+	infos = append(infos, streamingSegmentInfos...)
+
```
Comment (xiaofan-luan) [inline internal/querynode/impl.go:501]: print. a log about successufully get SegementInfo, how many historical segments, how many sealed segments blah blah

File: internal/querycoord/cluster.go [hunk 2]
```diff
@@ -376,6 +378,21 @@ func (c *queryNodeCluster) getSegmentInfo(ctx context.Context, in *querypb.GetSe
...
+
+	if node, ok := c.nodes[nodeID]; ok {
+		res, err := node.getSegmentInfo(ctx, in)
+		if err != nil {
+			return nil, err
+		}
+		return res.Infos, nil
+	}
+
+	return nil, errors.New("ReleasePartitions: can't find query node by nodeID")
```
Comment (xiaofan-luan) [inline internal/querycoord/cluster.go:393]: Error message is weird, NodeID should. be printed

File: internal/querynode/impl.go (hunk 1 above)
Comment (xige-16) [reply internal/querynode/impl.go]: fixed

File: internal/querycoord/cluster.go (hunk 2 above)
Comment (xige-16) [reply internal/querycoord/cluster.go]: fixed

**Pull request merged**